css/contact.css

```diff
@@ -0,0 +1,251 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+body {
+  margin: 0;
+  padding: 0;
+}
+h1,
+h2,
+p {
+  font-family: "Roboto";
+}
+h1 {
+  font-size: 100px;
+}
+/* HEADER */
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: white;
+  color: black;
+  display: flex;
+  align-items: center;
+  height: 120px;
+  z-index: 3;
+}
+
+.header .left {
+  flex: 1;
+  padding-left: 20px;
+}
+.left {
+  display: flex;
+  align-items: center;
+}
+.left p {
+  margin-right: 10px;
+}
+#box {
+  background-color: #0150fd;
+  height: 20px;
+  width: 20px;
+}
+
+.header .right {
+  flex: 1;
+  padding-right: 20px;
+  text-align: right;
+}
+
+.header .right ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.header .right ul li {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.header .right ul li:first-child {
+  margin-left: 0;
+}
+
+.header .right ul li a {
+  color: black;
+  text-decoration: none;
+}
+.header .right ul li a:hover {
+  color: #0150fd;
+}
+
+/* MAIN */
+
+main {
+  background-color: #e6dacd;
+  height: 1100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+#main-head {
+  display: flex;
+  align-items: center;
+  margin-top: 200px;
```

Comment:
Rather than have a different line for margin-top, margin-left, margin-bottom and margin-right values; you can break these up into just one line.

So here rather than have "margin-top: 200px", "margin-bottom: 15px" and "margin-left: 240px" on different lines of code, you could try just one line that says "margin: 200px 0 15px 240px", this way you make the CSS functions slightly shorter and easier to read, hope it helps you!